code-reviews/pull-requests.md

```diff
@@ -0,0 +1,30 @@
+# Pull Requests
+
+Changes to any main codebase - master branch in Git repository, for example - must be done using pull requests (PR).
+
+Pull requests enable:
+* Code inspection - see [Code Reviews](./README.md)
+* Running automated qualification of the code
+    * Linters
+    * Compilation
+    * Unit tests
+    * Integration tests etc.
+
+The requirements of pull requests can and should be enforced by policies, which can be set in the most modern version control and work item tracking systems. See [Evidence and Measures section](./README.md#evidence-and-measures) on Code Reviews page for more information.
+
+## General Process
+
+1. Implement changes based on the well-defined description and acceptance criteria of the task at hand
+1. Then, before creating a new pull request:
```

Comment:
Can we also add  linting before creating a new PR

Reply:
Done